src/StickyTableHeader.ts

```diff
@@ -83,8 +86,9 @@ export default class StickyTableHeader {
       let containerRect = this.tableContainer.getBoundingClientRect();
       let cloneRect = this.cloneContainer.getBoundingClientRect();
       const bodyRect = document.body.getBoundingClientRect();
-      const currentScroll = window.scrollY;
-      window.scrollTo({ top: containerRect.y - bodyRect.y - this.getTop() - 60 });
+      const currentScroll = this.scrollContainer ? this.scrollContainer.scrollTop : window.scrollY;
+      const scrollElement = this.scrollContainer ?? window;
```

Comment:
Good catch. I do believe this is the cause in the issue reported that you linked as well.

However we can be more smart with this and make it work seamlessly. Let's try to use a variation of this first response https://stackoverflow.com/questions/35939886/find-first-scrollable-parent

The variation should be to return the window in case we don't find an element. Or maybe it might work with body which might be found? Needs to be tested.

I recommend you create a manual test case under test.

Reply:
@annavik do you think you can continue on this PR or should I take over?